### ISSUE TITLE
Calamares does not open on znver1 isos

### ISSUE THREAD

**Issue by benbullard79** [OpenMandrivaAssociation/distribution#2867]
**OpenMandriva version**:
Cooker and possibly ROME

**Describe the bug**:
After updating to `grub2 2.06-4` to fix the issue in bug report: https://github.com/OpenMandrivaAssociation/distribution/issues/2845 now the znver1 iso's do not open.

**Steps to reproduce**:
Down load iso # 1610 or 1612 and boot and try to open Calamares.

**Observed behavior**:
![Screenshot_20230103_010749](https://user-images.githubusercontent.com/5704386/210290246-3334fa6d-cb83-459e-b153-0542c389b87a.jpg)

**Expected behavior**:
That the installer opens and allows user to install OMLx. 

**Additional comment**:
I tested 2 x86_64 iso's # 1609 & 1611 and both work. Then tested 2 znver1 isos # 1610 & 1612 and both are broken as described.

**Logs and screenshots if relevant**
Screen shot above.

[calamares-install-znver1.log](https://github.com/OpenMandrivaAssociation/distribution/files/10333973/calamares-install-znver1.log)

Calamares seems unhappy about: `ERROR: Module "bootloader" requires "partition" before it in sequence.`.

No clue why this happens on znver1 iso's and not on x86_64 iso's.

**Comment by benbullard79** [comment]
Removed ROME/rolling label. This bug seem to be Cooker znver1 iso's only.

I just installed ROME znver1 iso # 1614. With `f2fs` for /.

**Comment by tpgxyz** [comment]
This shouyld be fixed with new distro-release-23.90-1 - yes, same release ver.

**Comment by benbullard79** [comment]
I made new Cooker slim znver1 and x86_64 iso's and tested. Both worked so this bug is probably fixed. I am waiting on "regular" Cooker Plasma znver1 and x86_64 iso's to build. When I test those, if successful, I will close this bug report. Just my attempt to be thorough on this important bug fix.

FWIW: I used "Erase disk" with `f2fs` with both of the Cooker slim iso's. Works just fine. 

The Cooker slim iso's tested are #'s 1617, and 1618.
